### ISSUE TITLE
Can Stimfit detect positive-going events?

### ISSUE THREAD

**Issue by csong126** [neurodroid/stimfit#58]
Hello,

I haven't tried the software yet but I am wondering if it can detect positive-going spontaneous events, such as sIPSC? 

Thanks!

**Comment by schloegl** [comment]
Yes, you can also detect positive-going spontaneous event, you just need to select some example events, and define it as a template, and then apply some of the detection methods. 

Yes, Stimfit can also read some the heka patchmaster format - notable the v2 format. If you have data in Heka v1 format, you need to open the data with Patchmaster, and save it again, which will convert it to Heka v2 format. This will then be readable.
